repaygithub/cactus
# CACTUS-665 :: Require Media Queries

Ticket: https://repayonline.atlassian.net/browse/CACTUS-665

### Testing
1. Pull down this branch and run `yarn cleanup && yarn build && yarn web start`
2. Inspect some components that use breakpoints/media queries at different screen sizes and make sure they're behaving as they should at each respective screen size. Some good components to test with are:
  - Layout
  - BrandBar
  - Footer
  - Table
  - DataGrid
  - Basically any component that adjusts its display/layout depending on screen size.
